assignments/objects.js

```diff
@@ -18,21 +18,55 @@ const example = {
 }
 
 // Write your intern objects here:
+const int1 = {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmeloy0@psu.edu",
+    "gender": "F"
+}
+
+const int2 = {
+    "id": 2,
+    "name":"Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M"
+}
 
+const int3 = {
+    "id": 3,
+    "name": "Keven",
+    "email": "kmurrey2@wikimedia.org",
+    "gender": "M"
+    
+}
+
+const int4 = {
+    "id": 4,
+    "name": "Gannie",
+    "email": "gmartinson3@illinois.edu",
+    "gender": "M"
+}
+
+const int5 = {
+    "id": 5,
+    "name": "Antonietta",
+    "email": "adaine5@samsung.com",
+    "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(int1.name)= "Mitzi"
```

Comment:
We don't do equal signs `(=)` next to our console logs, JavaScript doesn't work like that